src/app/board/board.controller.js

```diff
@@ -1,10 +1,14 @@
 export class BoardController {
-  constructor ($firebaseArray, $location) {
+  constructor ($scope, $firebaseArray, $location) {
```

Comment:
I wasn't working on the board controller actually. This probably had something to do with like or edit feature. Will ask and correct the code.